### PR TITLE
Fixes #874 : Storing data of responses in ember data

### LINKED ISSUE #874: Storing data of responses in frontend
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behaviour:**

Whenever the file or an asset is saved in backend an id for the same is sent.

**Expected behaviour:**

The ids should be stored in a way that will then be used for the badge generation.

__Yes__ I am working on the same.

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes #874

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

- Added CSV Upload class according to JSON API
- Storing responses from the backend for badge generation

__Text data Upload__
image

__CSV Upload__

image

### CODE REVIEW

File: frontend/app/controllers/create-badges.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default Controller.extend({
       let csv_ = this.get('store').createRecord('csv-file', {
         uid,
         csvFile   : csvData,
-        extension : '.csv'
+        extension : 'csv'
```
Comment (gabru-md) [inline frontend/app/controllers/create-badges.js]: just curious, but is this not causing upload error on backend?

Comment (vaibhavsingh97) [reply frontend/app/controllers/create-badges.js]: I have the same question

Comment (yashLadha) [reply frontend/app/controllers/create-badges.js]: I don't think so as it is working and tested also refer : https://github.com/fossasia/badgeyay/blob/fc7ac4e999d1c3b9417d7e5af4ddc19a9ace64b1/api/controllers/fileUploader.py#L66

Comment (gabru-md) [reply frontend/app/controllers/create-badges.js]: Oh! It has been corrected before. Sorry! I did not see that/
:+1:

**Comment by yashLadha** [comment]
Please merge it @djmgit as i have other pr ready :sweat_smile:

**Pull request merged**